### DISCUSSION TITLE
Demo Changes in Student Dashboard

### DISCUSSION THREAD

**Discussion by fakhruddin-vsbizz** [priyadarshi0811/arabic-moallema-refactored-v2.0#59] [Announcements]
Demo change for student panel

**Comment by fakhruddin-vsbizz** [comment]
Teacher: Join Class - remove text field and insert label with meet link

**Reply by somesh2001** [reply]
Remove the input field and set the label for the URL

![image](https://user-images.githubusercontent.com/56119880/225858145-ff774c37-cf07-4b4f-b068-6f917fa26226.png)

![image](https://user-images.githubusercontent.com/56119880/225858321-e649ee11-3e2b-4820-b199-98aa6a13c8db.png)

**Comment by fakhruddin-vsbizz** [comment]
Activity - Last activity when submitted, either fixing the route or run `window.location.href = "localhost:3000/student/activity"`.

[ HIGH PRIORITY ]

**Reply by somesh2001** [reply]
Added window.location.href

![image](https://user-images.githubusercontent.com/56119880/225854859-28285d78-655e-447c-a45c-81b9555db2d7.png)

**Reply by fakhruddin-vsbizz** [reply]
Does it work as expected after submission?

**Comment by fakhruddin-vsbizz** [comment]
Learning: Module and Activity - Give back to dashboard button on each page.

[ HIGH PRIORITY ]

**Reply by atif-vsbizz** [reply]
Done
